music21/stream/__init__.py

```diff
@@ -8335,7 +8313,6 @@ def scaleDurations(self, amountToScale, *, inPlace=False):
         for e in returnObj.recurse().getElementsNotOfClass('Stream'):
             e.duration = e.duration.augmentOrDiminish(amountToScale)
 
-        returnObj.coreElementsChanged()
```

Comment:
Good.  Just putting a comment here where I can:  looking at the line 8322/8300 "We do not retain durations in any circumstance; if inPlace=True, two deepcopies are done" should be if "inPlace=False, two deepcopies of each duration are done" -- though actually deepcopying a duration is no longer slow, and I know an easy speedup since the v2.0 duration model.  (But no one uses this!)